src/Form/WmContentContainerForm.php

```diff
@@ -91,18 +90,10 @@ public function form(array $form, FormStateInterface $form_state)
             '#description' => t('Allowed bundles in this type.'),
         ];
 
-        $host_bundles = [];
-        if ($entity->getHostEntityType()) {
-            $host_bundles = $this->getAllBundles($entity->getHostEntityType());
-        } elseif (isset($values['host_entity_type'])) {
```

Comment:
@brownrl Is it ok that we don't look in $values here anymore? Why were we doing this?